src/EnergyPlus/Fans.cc

```diff
@@ -1343,8 +1343,7 @@ void FanComponent::set_size(EnergyPlusData &state)
     static constexpr std::string_view routineName = "FanComponent::set_size()"; // include trailing blank space
 
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-    bool _bPRINT = true;  // TRUE if sizing is reported to output (eio)
-    int NumFansSized = 0; // counter used to deallocate temporary string array after all fans have been sized
+    bool _bPRINT = true; // TRUE if sizing is reported to output (eio)
```

Comment:
[src/EnergyPlus/Fans.cc:849]:(style),[shadowVariable],Local variable 'cCurrentModuleObject' shadows outer variable
[src/EnergyPlus/Fans.cc:1347]:(style),[unreadVariable],Variable 'NumFansSized' is assigned a value that is never used.